<launch>

  <arg name="bag" doc="path to bag to be replayed"/>

  <node pkg="infuse_debug_tools" name="asn1_bitstream_to_tf" type="asn1_bitstream_to_tf_node" output="screen">

    <param name="publish_asn1_time" value="false" />
    <param name="virtual_frame" value="LocalTerrainFrame LAASParking -745.395003 375.152327 -1.494203 0 0 0" />
    <rosparam param="virtual_frames">
        ['LocalTerrainFrame LAASParking -745.395003 375.152327 -1.494203 0 0 0',
         'RoverBodyFrame RoverBaseFrame 0 0 0 0 0 0']
    </rosparam>
    <param name="topics_to_connect" value="/pose_robot_pom /infuse_crisp/sensorPoseOutput/Velodyne" />
    
  </node>

  <node pkg="rosbag" name="rosbag_play" type="play" output="screen" args="-d 5 $(arg bag)" />

  <node pkg="rviz" name="rviz" type="rviz" />

</launch>
